src/Core__List.resi

```diff
@@ -23,129 +23,158 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA. */
 
 /**
-  Collection functions for manipulating the `list` data structures, a singly-linked list.
+Collection functions for manipulating the `list` data structures, a singly-linked list.
 
-  **Prefer Array** if you need any of the following:
+**Prefer Array** if you need any of the following:
 
-  - Random access of element
-  - Better interop with JavaScript
-  - Better memory usage & performance.
+- Random access of element
+- Better interop with JavaScript
+- Better memory usage & performance.
 */
 /** `'a t` is compatible with built-in `list` type */
 type t<'a> = list<'a>
 
 /**
-  Returns the length of a list.
+`length(list)` returns the length of `list`.
 
-  ```res example
-  List.length(list{1, 2, 3}) // 3
-  ```
+## Examples
+
+```rescript
+List.length(list{1, 2, 3}) // 3
+```
 */
 let length: t<'a> => int
 
-/** **See** [`length`](##length) */
+/**
+`size(list)`. **See** [`length`](##length)
+
+## Examples
+
+```rescript
+List.size(list{1, 2, 3}) // 3
+```
+*/
 let size: t<'a> => int
 
 /**
-  Returns `Some(value)` where `value` is the first element in the list, or
-  `None` if `someList` is an empty list.
+`head(list)` returns `Some(value)` where `value` is the first element in the
+list, or `None` if `list` is an empty list.
+
+## Examples
 
-  ```res example
-  List.head(list{}) // None
-  List.head(list{1, 2, 3}) // Some(1)
-  ```
+```rescript
+List.head(list{}) // None
+List.head(list{1, 2, 3}) // Some(1)
+```
 */
 let head: t<'a> => option<'a>
 
 /**
-  Same as [head](#head), but raises an exception if `someList` is empty. Use
-  with care.
+`headExn(list)` same as [`head`](#head), but raises an exception if `list` is
```

Comment:
Should we add the same `## Exceptions` section here that you added in another PR?

Reply:
Done